recipes/nss/all/conanfile.py

```diff
@@ -0,0 +1,204 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os, glob
+
+
+class NSSConan(ConanFile):
+    name = "nss"
+    license = "Mozilla Public License"
```

Comment:
```suggestion
    license = "MPL-2.0"
```

https://spdx.org/licenses/MPL-2.0.html